src/frame/content.rs

```diff
@@ -36,6 +36,14 @@ impl Content {
         }
     }
 
+    /// Returns split values of the `Text` frame or None if the value is not `Text`.
+    pub fn text_values(&self) -> Option<Vec<&str>> {
```

Comment:
`split` is an iterator. Don't you think it would be nice to return `Option<impl Iterator<Item=&str>>` instead? This defers the decision to aggregate the values to the caller.